projects/kit/components/line-clamp/line-clamp-box.component.ts

```diff
@@ -5,7 +5,6 @@ import {PolymorpheusOutlet} from '@taiga-ui/polymorpheus';
 
 @Component({
     standalone: true,
-    selector: 'tui-hint',
```

Comment:
You still need some selector, otherwise it will be `ng-component` which we don't want